es-core/src/components/VideoComponent.cpp

```diff
@@ -27,7 +27,7 @@ void writeSubtitle(const char* gameName, const char* systemName, bool always)
 	int end = (int)(Settings::getInstance()->getInt("ScreenSaverSwapVideoTimeout") / (1000));
 	if (always) {
 		fprintf(file, "1\n00:00:01,000 --> 00:00:");
-		fprintf(file, std::to_string(end).c_str());
+		fprintf(file, "%s", std::to_string(end).c_str());
 		fprintf(file, ",000\n");
```

Comment:
Please can you leave the printfs as @pjft is going to re-do it (this could be a single line for example)